### PR TITLE
Filtering databases on databases listing, Issue #12047

### PR DESCRIPTION
**Pull request by 007durgesh219**
Before submitting pull request, please check that every commit:
- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them

Signed-off-by: Durgesh 007durgesh219@gmail.com

### CODE REVIEW

File: js/server_databases.js [hunk 1]
```diff
@@ -130,4 +131,56 @@ AJAX.registerOnload('server_databases.js', function () {
...
+        $('#tableFilter').hide();
+    }
+
+    var $filterField = $('#filterText');
+    /* Event handler for database filter */
+    $filterField.keyup(function (){
+       var textFilter = null, val = $(this).val();
+       if(val.length != 0) {
+           try {
+               textFilter = new RegExp("(^| )" + val.replace(/_/g, ' '), 'i');
```
Comment (nijel) [inline js/server_databases.js]: Why do you require start of string or space? I thing substring match would be more useful here.

**Comment by 007durgesh219** [comment]
Actually I copied from variable filter.  I Will change it

**Comment by 007durgesh219** [comment]
@nijel I've done suggested changes. Kindly review it again

**Comment by nijel** [comment]
Looks good, thanks1

**Pull request merged**